[docs] Add expand cell renderer (#1070)
mui-org/material-ui-x: docs/src/pages/components/data-grid/rendering/rendering.md

{{"demo": "pages/components/data-grid/rendering/RenderCellGrid.js", "defaultCodeOpen": false, "bg": "inline"}}

> #### Expand Cell Renderer

Reviewer: ```suggestion
#### Expand cell renderer
```

> By default, the grid will cut the content of a cell and render an ellipsis if the content of the cell does not fit in the cell.

Reviewer: ```suggestion
By default, the grid cuts the content of a cell and renders an ellipsis if the content of the cell does not fit in the cell.
```

> As workaround, you can create a cell renderer that will allow seeing the full content of the cell in the grid.

Reviewer: ```suggestion
As a workaround, you can create a cell renderer that will allow seeing the full content of the cell in the grid.
```
